src/time.cpp

```diff
@@ -100,11 +100,10 @@ namespace g3 {
       ts->tv_nsec = now.tv_usec * 1000;
       return 0;
 #elif(defined(WIN32) || defined(_WIN32) || defined(__WIN32__))
```

Comment:
please note that this will clash with your time pull request. Once that is merged in, which should be soon, please remember to update this PR.